tests/test_ML_helper.py

```diff
@@ -4,10 +4,11 @@
 from mqt.bench.utils import qiskit_helper
 
 from mqt.predictor import utils
+from mqt.predictor.ml import helper
```

Comment:
similar story as before. instead of these `_ML_` additions, just put the tests into a separate folder `tests/ml`.

Reply:
done.